docs/sftt/sftt-api-spec.md

```diff
@@ -1681,3 +1681,92 @@ If the site id specified is invalid.
 ##### `401 UNAUTHORIZED`
 
 If the calling user is not an admin.
+
+### Record Stewardship Activity
+
+`POST api/v1/protected/sites/:site_id/record_stewardship`
+
+Records a stewardship activity for the given site. Indicate `True` if the activity (watered, mulched, etc.) was completed, else `False`.
+
+#### Request Body
+
+```json
+{
+  "date": TIMESTAMP,
+  "duration": INT | NULL,
+  "watered": BOOLEAN,
+  "mulched": BOOLEAN,
+  "cleaned": BOOLEAN,
+  "weeded": BOOLEAN
```

Comment:
💭 We should probably check that these aren't all false. Could be done on the frontend

Reply:
Do we want to do this on the back end too? Feels like it should be done on the front end but I'm not sure if it's good to also require on the back end. @liammoyn

Reply:
Yeah we should do it on both. Lets specify it here so we don't forget during implementation, could throw a 400 if all are false

Reply:
Fixed